### ISSUE TITLE
feat(fiscal year): load the opening balance for a fresh installation of bhima

### ISSUE THREAD

**Issue by mbayopanda** [third-culture-software/bhima#3033]
This issue is about to add a way to : 
- Load opening balance of accounts for a fresh installation of bhima

The opening balance must be loded from a csv file or directly inserted for each accounts in bhima.

**Comment by jniles** [comment]
So, if I understand correctly:

This would write directly into period 0 of the `period_total` table for a fiscal year?

**Comment by mbayopanda** [comment]
Yes, but we can have this option just for the first fiscal year of any organization (in other word, for a fresh installation of bhima in any organization).
